bioptim/optimization/non_linear_program.py

```diff
@@ -139,6 +139,7 @@ def __init__(self):
         self.plot = {}
         self.q = None
         self.qdot = None
+        self.fatigable = None
```

Comment:
I think we have to specify what is `fatigable` since it can be muscles, tau, etc.

Reply:
I renamed fatigable to fatigable_tau which is a dict with three keys : fatigable_ma, fatigable_mr, fatigable_mf.

Reply:
This is no longer relevant since a lot has changed since!